src/cmd/CMakeLists.txt

```diff
@@ -1,3 +1,106 @@
+# Create model executable
+set(model_executable gz-sim-model)
+add_executable(${model_executable} model_main.cc ModelCommandAPI.cc)
+target_link_libraries(${model_executable}
+  gz-utils${GZ_UTILS_VER}::cli
+  ${PROJECT_LIBRARY_TARGET_NAME}
+)
+
+install(
+  TARGETS ${model_executable}
+  DESTINATION ${CMAKE_INSTALL_LIBEXECDIR}/gz/${GZ_DESIGNATION}${PROJECT_VERSION_MAJOR}/)
+
+# Build the unit tests.
+set(test_sources)
+
+# gz_TEST and ModelCommandAPI_TEST are not supported with multi config
+# CMake generators, see also cmd/CMakeLists.txt
+get_property(GENERATOR_IS_MULTI_CONFIG GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
+if(NOT GENERATOR_IS_MULTI_CONFIG)
```

Comment:
I think this is not needed for ModelCommandAPI since we're using the `$<CONFIG>` variable below which indicates that multi config generators are actually supported. If you move the `ModelCommandAPI_TEST` logic out of the if block and things fail, could you just add a TODO so we can come back to it later?